### PR TITLE
Signin page is responsive

### PR DESCRIPTION
**Pull request by dkap937-209**
Responsive to different screen widths
image
image

### CODE REVIEW

**Review by noahjade — APPROVED** [review]
Looking good 😎 Thanks for fixing up this styling!

**Review by ckar364 — COMMENTED** [review]
Great work! Any reason why the "position: center;" in the CSS file (WelcomeBackText class, approx. Line 26, 28) is commented out?

**Review by lin8231 — DISMISSED** [review]
Amazing, thanks for the changes! One small thing - could you please add cursor: pointer to SignInButton?

**Comment by dkap937-209** [comment]
> Great work! Any reason why the "position: center;" in the CSS file (WelcomeBackText class, approx. Line 26, 28) is commented out?

I had that before I made it responsive, I have removed it from the code now

**Review by noahjade — APPROVED** [review]
LGTM!

File: forgettable-frontend/src/pages/SignInPage/SignInPage.module.css [hunk 1]
```diff
@@ -36,9 +38,9 @@
     height: 60px;
     font-size: 20px;
```
Comment (ckar364) [inline forgettable-frontend/src/pages/SignInPage/SignInPage.module.css:39]: This font size is hard-coded, do we want to use an existing font size from global CSS?

Comment (lin8231) [reply forgettable-frontend/src/pages/SignInPage/SignInPage.module.css:39]: +1

**Review by noahjade — DISMISSED** [review]
LGTM!! :tada:

**Review by ckar364 — DISMISSED** [review]
LGTM

**Review by noahjade — APPROVED** [review]
Thanks for fixing the conflict!!

**Review by cdra334 — COMMENTED** [review]
Great work!

**Review by cdra334 — APPROVED** [review]
Oops forgot to approve

**Pull request merged**